src/test/java/org/zalando/nakadi/controller/EventTypeControllerTestCase.java

```diff
@@ -93,7 +93,7 @@ public void init() throws Exception {
 
         final NakadiSettings nakadiSettings = new NakadiSettings(0, 0, 0, TOPIC_RETENTION_TIME_MS, 0, 60,
                 NAKADI_POLL_TIMEOUT, NAKADI_SEND_TIMEOUT, 0, NAKADI_EVENT_MAX_BYTES,
-                NAKADI_SUBSCRIPTION_MAX_PARTITIONS, "service", "nakadi");
+                NAKADI_SUBSCRIPTION_MAX_PARTITIONS, "service", "nakadi", "");
```

Comment:
maybe you can set it here to some value and write a test in EventTypeControllerTest to check that header is actually set?

Reply:
@v-stepanov b09c4f3